modules/account/src/main/kotlin/edu/stanford/spezi/module/account/manager/AuthenticationManager.kt

```diff
@@ -78,7 +78,7 @@ internal class AuthenticationManager @Inject constructor(
                     "gender" to selectedGender,
                     "dateOfBirth" to dateOfBirth
                 )
-                firestore.collection("users").document(user.uid).set(userMap).await()
+                firestore.collection("patients").document(user.uid).set(userMap).await()
```

Comment:
@pauljohanneskraft, @Supereg: We will need to eventually clearly define the keys for user information stored in Spezi Account / SpeziFirebase in a documentation (e.g. inline docs or README) and we need to sync that up between iOS & Android.

@Basler182 Currently email and names are not stored in firebase as they are stored in firebase Authentication, we should do that the same way here: https://firebase.google.com/docs/auth/android/manage-users#get_a_users_profile

In addition on iOS we store gender & dob using the following keys:
- `DateOfBirthKey`. Date of both needs to be a Firebase datetime.
- `GenderIdentityKey`. Is stored as a string.
We should document this in the Firebase repo @pauljohanneskraft.

I think both could be improved in the future, that might be an open ticket and something to consider in the future @Supereg & @nriedman.

Reply:
@PSchmiedmayer  I think it would make sense to omit the ‘Key’ at the end of the field names and start with a lower case letter as we do in the other places as far as I know. I can update the email at this point as a precaution. We save the names in the user profile at this point anyway. I'll remove the double save in the Firestore. 

For the gender, it would be good if we had a list with the possible choices so that it is identical on iOS and Android. @pauljohanneskraft

Reply:
Sure, here is the list of supported types: https://github.com/StanfordSpezi/SpeziAccount/blob/e3344aaf5119e124d96aed39e04e4ce3f035ae41/Sources/SpeziAccount/Model/GenderIdentity.swift#L13

And I would leave the keys as they are for now; we should make this configurable & modifiable in a nice way in Spezi Account in the future @Supereg.

Reply:
Yes, that's correct. 00:00 at the time of the day is correct.